pyccel/ast/operators.py

```diff
@@ -559,6 +567,14 @@ def __new__(cls, arg1, arg2, simplify = False):
         if simplify:
             if isinstance(arg2, PyccelUnarySub):
                 return PyccelAdd(arg1, arg2.args[0], simplify = True)
+            elif isinstance(arg1, Literal) and isinstance(arg2, Literal):
+                dtype, precision = cls._calculate_dtype(arg1, arg2)
+                if dtype is NativeInteger():
+                    return LiteralInteger(arg1.python_value - arg2.python_value, precision=precision)
```

Comment:
See above. `PyccelUnarySub` may be needed

Reply:
done in [d289729](https://github.com/pyccel/pyccel/pull/797/commits/d289729fd8de96a2f0200dea18fbb6e770bbf690)